tests/frontend/verifier/test_lmeds.py

```diff
@@ -0,0 +1,23 @@
+"""Tests for frontend's LMEDS verifier.
+
+Authors: John Lambert
+"""
+import unittest
+
+import tests.frontend.verifier.test_verifier_base as test_verifier_base
+from gtsfm.frontend.verifier.lmeds import LMEDS
+
+
+class TestLmedsForFundamentalMatrix(test_verifier_base.TestVerifierBase):
+    """Unit tests for the LMEDS verifier w/o intrinsics in verification.
+
+    All unit test functions defined in TestVerifierBase are run automatically.
+    """
+
+    def setUp(self):
+        super().setUp()
+        self.verifier = LMEDS(use_intrinsics_in_verification=False, estimation_threshold_px=0.5)
```

Comment:
could you also test the case where use_intrinsics_in_verification = True?

Reply:
I actually found that setting (LMEDS w/ E-matrix estimation) yielded a result that couldnt pass the test error threshold, so I excluded it (`Angular error 9.3 vs. tol. 2.0`).

Reply:
even with data that was generated using perfect intrinsics? 
If that is the case, it sounds like it doesn't work. Maybe we should not submit that part if it doesn't work yet.

Reply:
That BaseVerifier unit test is checking a planar case (2 planes), which LMEDS - EMat is no good at.

But it works fine for Door : - ). I'll add a test to show that.
![Screen Shot 2022-07-23 at 1 16 15 AM](https://user-images.githubusercontent.com/16724970/180591552-195d79c8-b2fb-4ef6-bb5c-899a8dedde19.png)

Reply:
Actually, I'll add this extra test in a follow-up PR. (I've tested locally, but have a few other changes to APIs I want to push first).